CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
+# dbt_unified_rag v0.1.0-a6
+
+## Bug Fixes (requires `--full-refresh`)
+- Applied `coalesce_cast` macro to all relevant fields that are being concatenated into `comment_markdown`, as any concatenation in Snowflake with a null value returns null. We coalesced 'UNKNOWN' on a string field, and '1970-01-01 00:00:00' on a timestamp field to ensure Snowflake returns chunks of texts for all comments with null components.
+- Fields are now coalesced in these intermediate models:
+  - Hubspot
+    - `int_rag_hubspot__deal_comment_document`: `email_title` and `body` (string fields), `comment_time` (timestamp field).
+    - `int_rag_hubspot__deal_document`: `title` (string field) and `created_on` (timestamp field).
+  - Jira
+    - `int_rag_jira__issue_comment_document`: `comment_body` (string field) and `comment_time` (timestamp field).
+    - `int_rag_jira__issue_document`: `title` (string field) and `created_on` (timestamp field).
+  - Zendesk
+    - `int_rag_zendesk__ticket_comment_document`: `comment_body` (string field) and `comment_time` (timestamp field).
+    - `int_rag_zendesk__ticket_document`: `title` (string field) and `created_on` (timestamp field).
```

Comment:
Could we format this a lil differently? The current state sorta makes it sound like we're coalescing these fields _together_

I think this would be clearer with a markdown table or tbh even just splitting the fields into sub-bullets of their models. So like 
```yml
  - Hubspot
    - `int_rag_hubspot__deal_comment_document`: 
      - `email_title` (string)
      - `body` (string)
      - `comment_time` (timestamp)
```

Reply:
Sweet -- could you remove the code-block though? I myself am unclear why I made the suggestion in yml lol